utils.py

```diff
@@ -131,9 +109,55 @@ def divide_data(n_train, n_val):
             move(os.path.join(data_folder,test_image_dir), os.path.join(test_folder, os.path.basename(test_image_dir)))
     return train_folder, val_folder, test_folder
 
-train_folder, val_folder, test_folder = divide_data(35, 5)
+# function to combine all data preparation steps 
+def prepare_data(data_folder="data"):
+    """
+    :param string data_folder: folder for saving the data
+    :returns:
+        - train_folder - path to folder with subfolders for training data
+        - val_folder - path to folder with subfolders for validation data
+        - test_folder - path to folder with subfolders for testing data
+    """
+    os.makedirs(data_folder, exist_ok=True)
+    data_url = "https://zenodo.org/record/5092850/files/covid-if-groundtruth.zip?download=1"
+    download_url(data_url, os.path.join(data_folder, "data.zip"))
+    unzip(os.path.join(data_folder, "data.zip"), data_folder, remove=True)
+    file_paths = glob(os.path.join(data_folder, "*.h5"))
+    file_folders = convert_hdf5_to_tif(file_paths, file_folders)
+    train_folder, val_folder, test_folder = divide_data(35, 5, file_folders)
+    return train_folder, val_folder, test_folder
+
+
+if __name__ == "__main__":
```

Comment:
I would suggest not to duplicate the functionality here, but just call the `prepare_data` function.